packages/rum-core/src/performance-monitoring/breakdown.js

```diff
@@ -32,9 +32,12 @@ import { PAGE_LOAD } from '../common/constants'
  * Interested events from the Navigation timing API
  */
 const pageLoadBreakdowns = [
-  ['fetchStart', 'responseEnd', 'Network'],
-  ['domLoading', 'domInteractive', 'DOM Processing'],
-  ['domInteractive', 'loadEventEnd', 'Page Render']
+  ['domainLookupStart', 'domainLookupEnd', 'DNS'],
```

Comment:
Don't we want to include the time between `fetchStart` and `domainLookupStart`? The documentation suggests this time is used for checking application cache.

Reply:
AppCache https://www.html5rocks.com/en/tutorials/appcache/beginner/ is no more and its not a useful metric in this scenario and we don't need to account for duration of the whole transaction in the breakdown.